Source/WebKit/Shared/JavaScriptEvaluationResult.cpp

```diff
@@ -223,6 +224,15 @@ JavaScriptEvaluationResult::JavaScriptEvaluationResult(JSGlobalContextRef contex
 
 JSValueRef JavaScriptEvaluationResult::toJS(JSGlobalContextRef context, Value&& root)
 {
+    auto globalObjectForms = [] (auto context) {
```

Comment:
What do you mean by "forms"??

Reply:
In this case the JSGlobalObject*, JSDOMGlobalObject*, and Document* all basically point to the same thing.  They are different forms of the same global object.  I agree the name isn't great.  Maybe globalObjectPointers?